moveit_ros/moveit_servo/demos/cpp_interface/demo_twist.cpp

```diff
@@ -77,24 +77,35 @@ int main(int argc, char* argv[])
   // This is just for convenience, should not be used for sync in real application.
   std::this_thread::sleep_for(std::chrono::seconds(3));
 
+  // Initializing the target pose as end effector pose, this can be any pose.
+  auto robot_state = planning_scene_monitor->getStateMonitor()->getCurrentState();
+
+  // Get the robot state and joint model group info.
+  const moveit::core::JointModelGroup* joint_model_group =
+      robot_state->getJointModelGroup(servo_params.move_group_name);
+
   // Set the command type for servo.
   servo.setCommandType(CommandType::TWIST);
 
   // Move end effector in the +z direction at 10 cm/s
   // while turning around z axis in the +ve direction at 0.5 rad/s
-  TwistCommand target_twist{ "panda_link0", { 0.0, 0.0, 0.1, 0.0, 0.0, 0.5 } };
+  TwistCommand target_twist{ "panda_link0", { 0.0, 0.0, 0.05, 0.0, 0.0, 0.4 } };
```

Comment:
again, update the numbers in the comment above this line... or remove them.

I generally think it's a bad idea to copy-paste literal numbers in comments for this reason, so I'm pro removing.